bindings/js/swift_sharing_request_bind.js

```diff
@@ -19,6 +39,14 @@ class SwiftSharingRequest {
       this.address);
     url.searchParams.append("owner", owner);
 
+    let signed = await this._getSignature(
+      60,
+      "/request/user/".concat(username, "/", container)
```

Comment:
we should check the `.concat` does not include in the url anything that is not escaped